### PR TITLE
DPL-806 ONTFlowCell page restyling

### LINKED ISSUE #1244: DPL-806 Restyle ONTFlowCell page
User Story
As PSD we would like to ensure that the styling of Traction is consistent across pipelines.

Who are the primary contacts for this story
Steve / Seena

Acceptance criteria
To be considered successful the solution must allow:

-[ ] Redesign the flowcell style to make it more visually appealing and user friendly - improve the input boxes and labels, maybe make it look more like a real life flowcell?

### PR DESCRIPTION
**Pull request by seenanair**
Closes #1244 
Restyling of ONTFlowCell
 - Color change
 - Text size changes 
 -  Size change

### CODE REVIEW

File: package-lock.json
Comment (BenTopping) [inline package-lock.json:1]: Unstage this file. There was a recent node version increment so make sure you have the latest node version and install again.

**Review by BenTopping — CHANGES_REQUESTED** [review]
Definitely an improvement. I think the default border should be changed from white to something else as it makes the top of the flowcell look a bit off. I think the input field spacing is also slightly off, maybe because it leaves space for the tick/cross during validation, but it doesn't fill the width. This screenshot shows the input field spacing being slightly off and I was just playing around with some colours here like changing background to blue and setting border as grey. Screenshot 2023-08-07 at 10 43 20

File: src/components/ont/runs/ONTFlowcell.vue [hunk 1]
```diff
@@ -1,44 +1,53 @@
...
-            :classes="flowcell_barcode_field_colour"
-            placeholder="Scan library barcode"
-            :debounce="500"
-          />
-        </traction-field-error>
-      </fieldset>
+      <div class="text-xl p-2 text-gray-500 border-b-2">
+        {{ coordinate }}
+      </div>
+      <!-- <div class="border border-gray-300" /> -->
```
Comment (BenTopping) [inline src/components/ont/runs/ONTFlowcell.vue:11]: Can remove commented line

**Comment by seenanair** [comment]
> Definitely an improvement. I think the default border should be changed from white to something else as it makes the top of the flowcell look a bit off. I think the input field spacing is also slightly off, maybe because it leaves space for the tick/cross during validation, but it doesn't fill the width. This screenshot shows the input field spacing being slightly off and I was just playing around with some colours here like changing background to blue and setting border as grey. Screenshot 2023-08-07 at 10 43 20

Those colors look good, which one ypu used for blue, and border?

**Comment by BenTopping** [comment]
> Those colors look good, which one you used for blue, and border?
background: `bg-blue-100` (same as pacbio plates background)
border: `border-gray-300`
@seenanair

File: src/components/ont/runs/ONTFlowcell.vue (hunk 1 above)
Comment (seenanair) [reply src/components/ont/runs/ONTFlowcell.vue]: 👍 Done

**Comment by seenanair** [comment]
> > Definitely an improvement. I think the default border should be changed from white to something else as it makes the top of the flowcell look a bit off. I think the input field spacing is also slightly off, maybe because it leaves space for the tick/cross during validation, but it doesn't fill the width. This screenshot shows the input field spacing being slightly off and I was just playing around with some colours here like changing background to blue and setting border as grey. Screenshot 2023-08-07 at 10 43 20
> 
> Those colors look good, which one you used for blue, and border?

Done 👍

File: package-lock.json
Comment (seenanair) [reply package-lock.json:1]: Done 👍

File: src/components/shared/TractionInput.vue [hunk 2]
```diff
@@ -12,7 +12,7 @@
 -->
 
 <template>
-  <div class="flex flex-col" :class="$attrs.class">
+  <div class="flex flex-col w-full" :class="$attrs.class">
```
Comment (BenTopping) [inline src/components/shared/TractionInput.vue:15]: Worth checking if this affects the styling in any other places as we use this component a lot.

**Review by BenTopping — APPROVED** [review]
Looks really good! Big difference from before.

File: src/components/shared/TractionInput.vue (hunk 2 above)
Comment (seenanair) [reply src/components/shared/TractionInput.vue:15]: I had a quick look and it all seems fine. Please tell if you see something different.
I couldn't make the input to fill up the space in ONTFlowCell without doing this ( <slot> for <TractionFieldError>) .

**Comment by seenanair** [comment]
> Looks really good! Big difference from before.

Actually the real difference comes with the colours you suggested. Cool :)

**Comment by seenanair** [comment]
> Looks really good! Big difference from before.

The real difference came after changing colours and border you suggested! Cool! Thank you

File: src/components/shared/TractionInput.vue (hunk 2 above)
Comment (BenTopping) [reply src/components/shared/TractionInput.vue:15]: Should be able to bind use the vue3 class attribute binding. So adding `:class="w-full"` to the input fields you want to have at fall width. Not a problem though if it doesn't affect other places.

Comment (seenanair) [reply src/components/shared/TractionInput.vue:15]: I will check that as it seems to be a better option

Comment (seenanair) [reply src/components/shared/TractionInput.vue:15]: Tried it, strangely that's not changing anything

Comment (BenTopping) [reply src/components/shared/TractionInput.vue:15]: Ok no worries, I think its fine as is.

**Review by BenTopping — APPROVED** [review]

**Pull request merged**